src/api/status/public/js/build-log/check-for-build.js

```diff
@@ -3,11 +3,19 @@ import terminal from './terminal.js';
 import buildHeader from './build-header.js';
 import showToast from '../utils/toast.js';
 
+let build;
```

Comment:
Should we put this variable inside the function `checkBuildStatus()`?

Reply:
Putting it here is ok, we want to have a variable here containing the build being shown in the build logs. This is compared against upcoming builds, if the same, skip writing a new log to the terminal.

Reply:
I mean the best practice is to avoid global variable @humphd can I have your thought on this ?

Reply:
It's a module-level variable vs. global, so it's not terrible.  If it really bothered you, you could wrap this data + your function below in a class, so they go together.  But this is no worse than `showToast`, which is also being defined at the module level.

In general I agree with you: fewer variables, reduce their scope.  But this is hard to avoid completely.

Reply:
Understood !